### DISCUSSION TITLE
Filtering events

### DISCUSSION THREAD

**Discussion by jvrobert** [blakeblackshear/frigate#2793] [Ask A Question] [answered]
Is there a way to filter events by time? e.g. between xx:xx->yy:yy time/date? I'm literally becoming my own little surveillance state with people and cars coming and going, it's pretty hard to scroll through all that.

**Comment by NickM-27** [comment, marked as answer]
In the 0.10.0 beta there is a way to filter events by time

**Reply by jvrobert** [reply]
Nice, that's close - thanks. I'd also need time filtering which I didn't see (only date from what I can see), but maybe I'll try to add it in and push a PR.
